src/NearOrg/Notifications/Notification.jsx

```diff
@@ -338,7 +351,9 @@ return (
             />
           </Timestamp>
         </Text>
-        {/* <Desc>{desc}</Desc> */}
+        {actionable && actionable && contentDescription.text != null && (
```

Comment:
This looks much cleaner:
```suggestion
        {actionable && contentDescription?.text && (
```